src/tests/HDFFile.cxx

```diff
@@ -7,10 +7,33 @@
 #include "../helper.h"
```

Comment:
Is there really a clear-cut answer?  Both approaches have pros and cons.  Let's discuss, then document the policy somewhere.